spec/toy_robot_spec.rb

```diff
@@ -12,24 +12,32 @@
       expect(subject.y).to eq(0)
     end
 
-    specify 'that ToyRobot f is set to EAST' do
-      expect(subject.f).to eq(:east)
+    specify 'that ToyRobot facing is set to NORTH' do
```

Comment:
Is this part of the spec to face north on initialisation? I ask because of the code changed below.